src/main.jsx

```diff
@@ -1,18 +1,34 @@
-import * as React from 'react'
-import * as ReactDOM from 'react-dom/client'
+import React from 'react'
+import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import Home from './Components/Home.jsx'
-
 import './index.css'
 
+// const router = createBrowserRouter([
+//   {
+//     path: '/',
+//     element: <App>{ Home }</App> 
+//   },
+// ])
+
 const router = createBrowserRouter([
   {
     path: '/',
-    element: <App>{ Home }</App> 
+    element: <App />,
+    children: [
+      {
+        path: '/',
+        element: <Home />
+      }
+    ]
   },
 ])
 
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <RouterProvider router={router} />
+  <React.StrictMode>
+    <RouterProvider router={router} />
+  </React.StrictMode>,
 )
+// strict mode will help with state as components mount and unmount
+// source: https://refine.dev/blog/react-strict-mode-in-react-18/
```

Comment:
I have heard that strict mode isn't necessary so my inner teen rebelled. But the capacity for catching errors makes a lot of sense.